arm/Microsoft.KubernetesConfiguration/extensions/readme.md

```diff
@@ -0,0 +1,68 @@
+# KubernetesConfiguration Extensions `[Microsoft.KubernetesConfiguration/extensions]`
+
+This module deploys KubernetesConfiguration Extensions.
+
+## Prerequisites
+
+Registration of your subscription with the AKS-ExtensionManager feature flag. Use the following command:
+
+```powershell
+az feature register --namespace Microsoft.ContainerService --name AKS-ExtensionManager
+```
+
+Registration of the following Azure service providers. (It's OK to re-register an existing provider.)
+
+```powershell
+az provider register --namespace Microsoft.Kubernetes
+az provider register --namespace Microsoft.ContainerService
+az provider register --namespace Microsoft.KubernetesConfiguration
+```
+
+For Details see [Prerequisites](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/tutorial-use-gitops-flux2)
+
+## Navigation
```

Comment:
Move Navigation to the top.

It's also a bit odd that it did not automatically add it as a header... Can you re-run the script after the move?

Reply:
done